src/ui/layout/search/SearchResultsDropDown.vue

```diff
@@ -91,12 +104,18 @@ export default {
         previewChanged(changedPreviewState) {
             this.previewVisible = changedPreviewState;
         },
-        showResults(passedAnnotationResults, passedObjectResults) {
-            if ((passedAnnotationResults && passedAnnotationResults.length)
-                || (passedObjectResults && passedObjectResults.length)) {
+        showSearchStarted() {
+            this.searchLoading = true;
+            this.resultsShown = true;
+            this.annotationResults = [];
+            this.objectResults = [];
+        },
+        showResults(searchLoading, passedSearchQuery, passedAnnotationResults, passedObjectResults) {
```

Comment:
should we use destructuring here to clean up the function signature?

Like this: https://github.com/ryanmcdermott/clean-code-javascript#function-arguments-2-or-fewer-ideally